### PR TITLE
fix(TextBlock): fix paragraphs margins

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Fix margin for `p` elements added by rich text rendering.

### CODE REVIEW

**Review by robinandeer — COMMENTED** [review]
Hmm... instead of overriding, can we apply the margin styles only to the places that need them? 🤔

**Comment by guilhermespopolin** [comment]
> Hmm... instead of overriding, can we apply the margin styles only to the places that need them? 🤔

The issue is that rich text styles are applied in a broad way: https://github.com/HedvigInsurance/racoon/blob/main/apps/store/src/components/RichText/RichText.styles.tsx#L131
I don't se how we can be more specific on the selector there based on the way it works 🤔

File: apps/store/src/components/RichText/RichText.styles.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export const listStyles = css({
       width: 15,
       height: 15,
       borderRadius: '100%',
-      backgroundColor: theme.colors.textSecondary,
+      backgroundColor: 'var(--color, currentColor)',
```
Comment (guilhermespopolin) [inline apps/store/src/components/RichText/RichText.styles.tsx:75]: Thanks for the tip with `currentColor` @gustaveen 🙌🏻

File: apps/store/src/components/RichText/RichText.styles.tsx [hunk 2]
```diff
@@ -116,43 +116,53 @@ export const listStyles = css({
...
+  lineHeight: 1.32,
+})
 
-    '.preamble': {
-      display: 'block',
-      marginBottom: theme.space.xl,
-      fontSize: theme.fontSizes.xl,
-      color: theme.colors.textPrimary,
-    },
+export const paragraphyStyles = css({
```
Comment (robinandeer) [inline apps/store/src/components/RichText/RichText.styles.tsx:124]: nit: spelling mistake -> `paragraphStyles `

**Review by robinandeer — APPROVED** [review]
This looks reasonable :D

File: apps/store/src/components/RichText/RichText.styles.tsx (hunk 2 above)
Comment (guilhermespopolin) [reply apps/store/src/components/RichText/RichText.styles.tsx]: Thanks

**Pull request merged**